DetectDynamicJS.py

```diff
@@ -77,7 +77,14 @@ def doPassiveScan(self, baseRequestResponse):
         statusCode = responseInfo.getStatusCode()
         if statusCode != 200:
           return
-          
+
+        # Check for authorization
+        reqHeaders = self._helpers.analyzeRequest(self._requestResponse).getHeaders()
+        hfields = [h.split(':')[0] for h in reqHeaders]
+        ifields = ['cookie','authorization']
+        if not any(h for h in ifields if h not in str(hfields).lower()):
+          return
```

Comment:
I like this check. But please use `return None` in this function. It is a style question. Also indentation again.